toolchain-model/My.toolchain

```diff
@@ -1,83 +1,83 @@
-          <queryCapabilities title="QueryCapability" label="QueryCapability" queryBaseURI="query" resourceTypes="_e8ivgJa4EeqoX6mSFj9log"/>
-        </services>
-      </serviceProviders>
-    </serviceProviderCatalog>
-    <webServices name="TestScriptService" serviceUrlPattern="testScripts" resourceUrlPattern="{testScriptId}" resourceTypes="_e8ivgJa4EeqoX6mSFj9log" delete="true" update="true"/>
-    <specification/>
-    <requiredAdaptors xsi:type="oslc4j_ai:GenericRequiredAdaptor" serviceProviderCatalogURI="http://localhost:8083/adaptor-rm/services/catalog/singleton" name="RequirementsAdaptor" servicedResources="_e8j9oJa4EeqoX6mSFj9log"/>
-    <configuration xsi:type="oslc4j_ai:MavenServerConfiguration" rootServerBaseUrl="http://localhost:8082/" applicationContextPath="/adaptor-testing" servletUrlPattern="/services/*" doNotRegenerateJspFiles="true" jettyPort="8082">
-      <generalConfiguration filesBasePath="../adaptor-testing-webapp/" javaBasePackageName="com.sample.testing"/>
-      <projectConfiguration xsi:type="oslc4j_ai:MavenProjectConfiguration" lyoVersion="5.0.0-SNAPSHOT" groupId="com.sample.testing" artifactId="testing-tool" version="0.0.1-SNAPSHOT"/>
-    </configuration>
-  </adaptorInterfaces>
-  <adaptorInterfaces name="RM Tool" backendCodeTemplate_getServiceProviders="" backendCodeTemplate_getResource="">
-    <serviceProviderCatalog title="Service Provider Catalog">
-      <serviceProviders title="Service Provider">
-        <services domainSpecification="//@specification/@domainSpecifications[name='Requirements%20Management']">
-          <creationFactories title="CreationFactory1" label="CreationFactory1" creationURI="create" resourceTypes="_e8j9oJa4EeqoX6mSFj9log"/>
-          <queryCapabilities title="QueryCapability" label="QueryCapability" queryBaseURI="query" resourceTypes="_e8j9oJa4EeqoX6mSFj9log"/>
-          <selectionDialogs title="SelectionDialog1" label="SelectionDialog1" dialogURI="selector" resourceTypes="_e8j9oJa4EeqoX6mSFj9log"/>
-          <creationDialogs title="CreationDialog1" label="CreationDialog1" dialogURI="creator" resourceTypes="_e8j9oJa4EeqoX6mSFj9log"/>
-        </services>
-      </serviceProviders>
-    </serviceProviderCatalog>
-    <webServices name="RequService" serviceUrlPattern="requirements" resourceUrlPattern="{requirementId}" resourceTypes="_e8j9oJa4EeqoX6mSFj9log" delete="true" update="true"/>
-    <trsService xsi:type="oslc4j_ai:InmemPagedTrsService" basePageLimit="50" changeLogPageLimit="50"/>
-    <specification/>
-    <requiredAdaptors xsi:type="oslc4j_ai:GenericRequiredAdaptor" serviceProviderCatalogURI="http://your.host/adaptor/services/catalog/singleton" name="GenericRequiredAdaptor"/>
-    <configuration xsi:type="oslc4j_ai:MavenServerConfiguration" rootServerBaseUrl="http://localhost:8083/" applicationContextPath="/adaptor-rm" servletUrlPattern="/services/*" doNotRegenerateJspFiles="true" generateJspFilesForOslcUI="true" jettyPort="8083">

```

Comment:
That's what is dangerous with fixed white-space changes at the same time. After disabling that, I see 2 issues:

1. doNotRegenerateJspFiles="true" is added! We should not so we can capture such changes in LyD.
2. I don't see the change to Lyo 5.0.0-SNAPSHOT!

Reply:
In some toolchain model I had to spent lots of time to layout everything again so I wanted to be sure we don't lose it. Maybe it was refimpl then.

I think do not generate was ON before me and I actually disabled it because JSP files were not getting generated. Are you saying it looks the other way around in the diff?